Assets/HoloToolkit/UX/Scripts/Progress/ProgressIndicatorStyleEnum.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace HoloToolkit.UX.Progress
+{
+    /// <summary>
+    /// Enum describing the style of the Indicator portion
+    /// </summary>
+    public enum IndicatorStyleEnum
```

Comment:
I'd actually recommend keeping it the way it is so that the parameters and Fields that use this Enum can be named IndicatorStyle.

Generally I use Enum or Type suffix for that reason.

Reply:
I can see that.